src/WPF/WPF.Viewer/Samples/Analysis/QueryFeatureCountAndExtent/QueryFeatureCountAndExtent.xaml

```diff
@@ -20,35 +20,21 @@
                     <RowDefinition Height="auto" />
                     <RowDefinition Height="auto" />
                 </Grid.RowDefinitions>
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*" />
-                    <ColumnDefinition Width="*" />
-                </Grid.ColumnDefinitions>
                 <TextBlock Grid.Row="0"
-                           Grid.Column="0"
-                           Grid.ColumnSpan="2"
                            Margin="5,0,5,0"
                            FontWeight="SemiBold"
-                           Text="Click 'Zoom to matching features' to zoom to features matching the given state abbreviation. Click 'Count features in extent' to count the features in the current extent (regardless of the results of the query)."
+                           Text="Select a state to zoom to matching features:"
                            TextWrapping="Wrap" />
-                <TextBox x:Name="StateTextbox"
-                         Grid.Row="1"
-                         Grid.Column="0"
-                         Grid.ColumnSpan="2"
-                         Text="NH" />
+                <ComboBox x:Name="StatesComboBox"
+                          Grid.Row="1"
+                          SelectionChanged="ZoomToFeatures" />
                 <Button Grid.Row="2"
```

Comment:
I'm nitpicking here but could the button and the `ComboBox` be the same width? The `ComboBox` is slightly larger and it looks odd.